content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/_index.md

```diff
@@ -520,7 +724,15 @@ To remove APM instrumentation and stop sending traces from a specific service, f
 2. Restart the service.
 {{% /tab %}}
 
-{{% tab "Kubernetes" %}}
+{{% tab "Kubernetes (Preview)" %}}
+
+#### Use workload selection (recommended)
+
+With workload selection, you can enable and disable tracing for specific applications. [See configuration details here](#advanced-options).
```

Comment:
I don't know if there's a way to link to the explicit panel that can open because when you go here it throws you back to the _top_ of the entire section and is a little confusing.